### ISSUE TITLE
Add validation check for prior references pattern

### ISSUE THREAD

**Issue by sambodeme** [GSA-TTS/FAC#2625]
This ticket is related to https://github.com/GSA-TTS/FAC/issues/2545 and is intended to ensure that intake audit findings prior references data match the expected [pattern](https://github.com/GSA-TTS/FAC/blob/main/backend/schemas/source/base/Base.libsonnet#L292)
```
PriorReferences: Types.string {
    title: 'PriorReferences',
    description: 'Prior references',
    pattern: '^20[1-9][0-9]-[0-9]{3}(,\\s*20[1-9][0-9]-[0-9]{3})*$',
  }
```

**Comment by sambodeme** [comment]
This will be closed by #2606

**Comment by sambodeme** [comment]
Closed by PR #2606
